common/text-level-semantics/text-level-semantics.html

```diff
@@ -0,0 +1,148 @@
+---
+{
+	"title": "Text level semantics",
+	"language": "en",
+	"altLangPage": "texte-niveau-semantique.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-07-06"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<div class="alert alert-warning">
+<p>The purpose of this page is to test all native content grouping related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization. The following include all content grouping elements in the HTML5 specification and a few examples was inspired by the <a href="https://html.spec.whatwg.org/multipage/forms.html#forms">WHATWG section 4.4</a> as February 2023. This page may not contain all the possible grouping content element combination.</p>
+</div>
+
+<h2>The <code>a</code> element</h2>
+<p>Hyperlinks</p>
```

Comment:
Need the anchor element in this example

```suggestion
<ul>
  <li><a>Anchor</a> without <code>href</code></li>
  <li><a href="#">Anchor with an empty URL fragment</a></li>
  <li id="local-anchor"><a href="#local-anchor>Anchor with a non-empty URL fragment</a></li>
  <li><a href="https://wet-boew.github.io/GCWeb/">Anchor with <code>https://<code> URL scheme</a></li>
  <li><a href="https://wet-boew.github.io/GCWeb/index-en.html" hreflang="en">Anchor with an English <code>hreflang</code> configuration</a></li>
  <li><a href="https://wet-boew.github.io/GCWeb/index-fr.html" hreflang="fr">Anchor with an French <code>hreflang</code> configuration</a></li>
</ul>
```